### DISCUSSION TITLE
Community Meeting time change!

### DISCUSSION THREAD

**Discussion by OrlinVasilev** [vmware-tanzu/velero#5091] [General discussions]
**Hello Community!**
Problem I'm looking to solve: The current setup of the community meetings is not optimal as we have two separate community meetings with different attendees, the current setup is two alternating meetings in two different time zones.

The challenge: we have to accommodate the following timezones: US EST, US PST, China: Beijin and UTC+3(Europe).
 
I think if we can make it bi-weekly in the only time slot that we all are awake it will be most productive!

If we keep it Tuesday : like this https://www.worldtimebuddy.com/?qm=1&lid=727011,1816670,8,5&h=727011&date=2022-7-5&sln=17-18&hf=2

<img width="821" alt="image" src="https://user-images.githubusercontent.com/7236111/177159105-8f5330b0-943c-40d0-963e-475cdcb4c7c5.png">

I suspect there is a chance to be inconvenient for the folks at PDT but let's have some discussion!

Other options is to make again in two sessions like:
Bi-weekly (Tue - EDT 9:00 AM / PDT 6:00 AM / BJ 9:00 or 10:00 PM)
Bi-weekly (Thu - BJ 8:00 AM / EDT 8:00 or 7:00 PM / PDT 5:00 or 4:00 PM PT)

Please share your thoughts here so we can make it work!

Orlix

**Comment by yanji09** [comment]
10pm is a bit too late for Beijing. And by the time daylight saving time switches again in November, it will be 11pm.  And it's also not friendly for PT time zone folks (7am), I personally think the second option - 2 separate biweekly schedules - might be more appropriate.

**Comment by euclidsun** [comment]
It seems better to have two session series...from time zone perspective.

One session will make the people hard to join from two locations (PDT and Beijing)

**Comment by sseago** [comment]
If we keep two time zones, does this mean we'd keep current times? Another possibility would be to keep the current Beijing time meeting as-is, but move the US-centric meeting two hours earlier (to the time proposed above). For the current schedule, the Beijing time is only able to attend twice a month (at 8 AM), as the US-centric meeting is at 12 or 1 AM depending on summer/winter time. If we were to move the US meeting only, this would mean that twice a month, it would be inconvenient but still possible for Beijing, in case someone really wants to join the meeting.

**Reply by yanji09** [reply]
The 2nd proposed option above keep two timezones. But with some changes.

1. change the twice a month to Bi-weekly to avoid the problem when the 1st meeting is Tuesday-based but the second is Wednesday-based. (if the 1st week start from Wed).
2. Move the US-centric meeting 2 hours earlier (Start from 10:00 AM). So that make it possible for BJ to join. but not that friendly for PT timezone.
3. Keep the BJ meeting same time at 8:00 AM.

So will look like below.
Bi-weekly (US Tue - EDT 10:00 AM / PDT 7:00 AM / BJ 10:00 or 11:00 PM)
Bi-weekly (BJ Wed - BJ 8:00 AM / EDT 8:00 or 7:00 PM / PDT 5:00 or 4:00 PM PT)

**Comment by weshayutin** [comment]
I do think it's important for the community to meet live on a periodic interval and every two weeks seems like a good pace to me.   As the velero community grows more geographically diverse I think async communication should become more important.  I don't see a lot of conversation, design discussion etc on the mailing list [1] at the moment.   I would highly encourage folks to more often leverage the mailing list to communicate than to save up items for the live community meeting.   This is the practice in other large open source projects.  It's almost rude not to be overly communicative on the mailing list, as you run the risk of leaving someone out who can not join live community sessions.  

[1] https://groups.google.com/g/projectvelero

**Reply by reasonerjt** [reply]
@weshayutin
Thanks for pointing this out.  Would you use the slack channels?
[velero-dev](https://kubernetes.slack.com/archives/C021GPR1L3S)
[velero-users](https://kubernetes.slack.com/archives/C6VCGP4MT)

**Comment by reasonerjt** [comment]
@OrlinVasilev Thanks for the proposal the new schedule LGTM!

**Comment by shubham-pampattiwar** [comment]
+1 the proposal looks good! @OrlinVasilev

**Comment by OrlinVasilev** [comment]
Tomorrow 19th of Oct will send the new invites and will cancel the old :)

**Comment by OrlinVasilev** [comment]
Thank you all! with the new invites  set I think we can close this one!
